mmv1/third_party/terraform/website/docs/r/apigee_shared_flow.html.markdown

```diff
@@ -0,0 +1,88 @@
+subcategory: "Apigee"
+page_title: "Google: google_apigee_shared_flow"
+description: |-
+  You can combine policies and resources into a shared flow that you can consume from multiple API proxies, and even from other shared flows.
+---
+
+# google\_apigee\_shared\_flow
+
+You can combine policies and resources into a shared flow that you can consume from multiple API proxies, and even from other shared flows. Although it's like a proxy, a shared flow has no endpoint. It can be used only from an API proxy or shared flow that's in the same organization as the shared flow itself.
+
+
+To get more information about SharedFlow, see:
+
+* [API documentation](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.sharedflows)
+* How-to Guides
+    * [Sharedflows](https://cloud.google.com/apigee/docs/resources)
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+* `name` -
+  (Required)
+  The ID of the shared flow.
+
+* `org_id` -
+  (Required)
+  The Apigee Organization associated with the Apigee instance,
+  in the format `organizations/{{org_name}}`.
+
+* `config_bundle` -
+  (Required)
+  The configuration bundle zip file path.
+
+- - -
+
+
```

Comment:
Can we add `md5hash` and `detect_md5hash` in their appropriate spots here?

Reply:
@megan07 md5hash and detect_md5hash are "imaginary/speculative" and "local only" attributes that is used to detect modifications on referenced file and not intended to be modified by TF user. This pattern is inspired from  [storage_bucket_object](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/resources/resource_storage_bucket_object.go#L118)
Let me know what you think :)

Reply:
sure! it appears they're still [documented for storage_bucket_object](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_bucket_object#detect_md5hash), so whether we want to remove there or add here, i'd advocate that we stay consistent. thanks for pointing that out though, that helps with context around those pieces!